src/subscription_manager/repofile.py

```diff
@@ -462,19 +462,28 @@ def sections(self):
 
         def fix_content(self, content):
             # Luckily apt ignores all Fields it does not recognize
-            baseurl = content["baseurl"]
+            parsed_url = urlparse(content["baseurl"])
```

Comment:
In my development environment, I had to change this to:
```suggestion
            parsed_url = urlparse(unquote(content["baseurl"]))
```
Maybe this is something that changed with a newer version of candlepin?

Perhaps we should make sure it will work both for quoted or unquoted baseurls!

Reply:
Using a recent Candlepin (4.4.x) by chance? If so, the problem should be a regression in CP, see also
https://community.theforeman.org/t/status-code-403-for-error-failed-to-download-metadata-for-repo-after-upgrade-to-3-11/38601/7
(CP developers are aware, unfortunately no public tickets to track this)